test/ngAnimate/animateSpec.js

```diff
@@ -292,6 +292,129 @@ describe("animations", function() {
       });
     });
 
+    describe('customFilter()', function() {
+      it('should clear the `customFilter` if no function is passed',
+        module(function($animateProvider) {
+          expect($animateProvider.customFilter()).toBeNull();
+
+          $animateProvider.customFilter(angular.noop);
+          expect($animateProvider.customFilter()).toEqual(jasmine.any(Function));
+
+          $animateProvider.customFilter({});
```

Comment:
I kept this consistent with `classNameFilter` (although it is preferrable to throw obviously).
Throwing would be a BC for `classNameFilter`. Should I fix this one to throw and make `classNameFilter` throw for 1.6 (or leave it as is)?